### ISSUE TITLE
Can't set max gate to zero in example setup program

### ISSUE THREAD

**Issue by svdasein** [ncmreynolds/ld2410#28]
In your example you have 

`if(newMovingMaxDistance > 0 && newStationaryMaxDistance > 0 && newMovingMaxDistance <= 8 && newStationaryMaxDistance <= 8)`

This seems to properly limit gate selection to a maximum of 8, but - gate 0 is a valid maximum isn't it?

**Comment by jenschr** [comment]
Yes. This is however more like a support question for [one](https://github.com/ncmreynolds/ld2410/blob/main/examples/setupSensor/setupSensor.ino) of the [two example files](https://github.com/ncmreynolds/ld2410/tree/main/examples), rather than an issue with the library?
